src/components/ui/Mark.tsx

```diff
@@ -23,26 +23,37 @@ const questionStyles = (small?: boolean, disabled?: boolean) =>
     cursor(disabled ? undefined : 'cursor-pointer')
   )
 
-const borderWrapper = (small?: boolean) =>
+const borderWrapper = (small?: boolean, medium?: boolean) =>
```

Comment:
This might grow later, some of us will add `big?:boolean, big-med, small-med etc.`, we need to think how to avoid it

Reply:
I tried to simplify it, don't have any workarounds in my mind